src/main/java/com/mtab/spring/boot/service/models/Book.java

```diff
@@ -0,0 +1,9 @@
+package com.mtab.spring.boot.service.models;
+
+public record Book(
+        String bookID,
+        String title,
+        String author,
+        Integer price,
```

Comment:
In most cases, it is better to use the primitive, `int` rather than the object.

It's a long story as to why, but trust me for now. Java can automatically convert between the primitive and the object (called "boxing" and "unboxing").